### PR TITLE
Update documentation for the Bolt Payment Method

### PR DESCRIPTION
**Pull request by piyushswain**
After a recent update, now Bolt has been added as an installable payment method during Solidus installation.

The doc has been updated to reflect the same information.

### CODE REVIEW

**Comment by waiting-for-dev** [comment]
Thanks! It looks good. We need to put it on hold until #4225 is merged. We might need to revisit if PayPal gets removed as an option (their SDK doesn't support Ruby 3).

**Comment by elia** [comment]
https://github.com/solidusio/solidus/pull/4425 has been closed, but https://github.com/solidusio/solidus_frontend/pull/14 has been merged, but PayPal has been since disabled.

I would suggest to merge anyway, knowing that 2.3 will restore it, @waiting-for-dev @piyushswain is that ok with you?

**Comment by waiting-for-dev** [comment]
Yeah. The edge guides track master, so let's wait until we have added the payment methods in core.

**Comment by elia** [comment]
I rebased fixing the conflict and back porting to 3.2

**Review by waiting-for-dev — APPROVED** [review]
Thanks, @piyushswain & @elia. I improved the text a bit.

`Next it will` => `Next, it will`
`with solidus` => `with Solidus`
`.Currently` => `. Currently`
`Solidus comes packaged` => `it comes packaged`

**Pull request merged**
